src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -0,0 +1,570 @@
+
+                currentValue = value;
+
+                UpdateCurrentValue();
+            }
+        }
+
+        /// <summary>
+        /// The max value of Picker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int MaxValue
+        {
+            get
+            {
+                return maxValue;
+            }
+            set
+            {
+                if (maxValue == value) return;
+                if (currentValue > value) currentValue = value;
+                
+                maxValue = value;
+                needItemUpdate = true;
+
+                UpdateValueList();
+            }
+        }
+
+        /// <summary>
+        /// The min value of Picker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int MinValue
+        {
+            get
+            {
+                return minValue;
+            }
+            set
+            {
+                if (minValue == value) return;
+                if (currentValue < value) currentValue = value;
+                
+                minValue = value;
+                needItemUpdate = true;
+
+                UpdateValueList();
+            }
+        }
+
+        /// <summary>
+        /// Applies style to Picker.
+        /// </summary>
+        /// <param name="viewStyle">The style to apply.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void ApplyStyle(ViewStyle viewStyle)
+        {
+            base.ApplyStyle(viewStyle);
+
+            //Apply StartScrollOffset style.
+            if (pickerStyle?.StartScrollOffset != null)
+                startScrollOffset = (int)pickerStyle.StartScrollOffset.Height;
+
+            //Apply ItemTextLabel style.
+            if (pickerStyle?.ItemTextLabel != null)
+            {
+                itemHeight = (int)pickerStyle.ItemTextLabel.Size.Height;
+
+                if (itemList != null)
+                    foreach (TextLabel textLabel in itemList)
+                        textLabel.ApplyStyle(pickerStyle.ItemTextLabel);
+            }
+
+            //Apply PickerCenterLine style.
+            if (pickerStyle?.Divider != null && upLine != null && downLine != null)
+            {
+                upLine.ApplyStyle(pickerStyle.Divider);
+                downLine.ApplyStyle(pickerStyle.Divider);
+                downLine.PositionY = (int)pickerStyle.Divider.PositionY + itemHeight;
+            }
+        }
+                
+        private void Initialize()
+        {
+            HeightSpecification = LayoutParamPolicies.MatchParent;
+
+            //Picker Using scroller internally. actually it is a kind of scroller which has infinity loop,
+            //and item center align features.
+            pickerScroller = new PickerScroller(pickerStyle)
+            {
+                Size = new Size(pickerStyle.Size.Width, pickerStyle.Size.Height),
```

Comment:
How about changing this as follows to show Picker with correct size when Picker size is set by user?

Size = new Size(-1, pickerStyle.Size.Height),

Reply:
I agree with your comment.